.github/workflows/release.yml

```diff
@@ -24,6 +24,18 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v6
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+          git_committer_name: "textileio-machine"
+          git_committer_email: "40302381+textileio-machine@users.noreply.github.com"
```

Comment:
the existing key we set up for our textileio bot looks to be valid. this repo has the matching GPG key configured, which should ensure the commit from `changesets` is verified